api/projects/models.py

```diff
@@ -61,6 +61,16 @@ class Project(LifecycleModelMixin, SoftDeleteExportableModel):
         max_length=255,
         help_text="Used for validating feature names",
     )
+    max_segments_allowed = models.IntegerField(
+        default=200, help_text="Max segments allowed for this project"
+    )
+    max_features_allowed = models.IntegerField(
+        default=400, help_text="Max features allowed for this project"
+    )
+    max_segments_overrides_allowed = models.IntegerField(
+        default=350,
+        help_text="Max segments overrides allowed for any (one) environment within this project",
+    )
```

Comment:
Would it be better to keep the environment variables and use those as the defaults? I'm not sure tbh, it might not be.

Reply:
I think we should also put these defaults back down to what we had previously (before I updated them yesterday), so 100 segments, 100 overrides, 400 features. We can then manually update these fields for the projects that currently have higher numbers. 

Could you verify here that these attributes all appear in the django admin?

Reply:
Yeah, I have added them